docs/make.jl

```diff
@@ -2,34 +2,40 @@ using Documenter, PowerSystems
 import DataStructures: OrderedDict
 using Literate
 
+include(joinpath(@__DIR__, "../src/utils/data.jl"))
+import .UtilsData: TestData
+download(TestData; branch = "master")
+
 # This is commented out because the output is not user-friendly. Deliberation on how to best
 # communicate this information to users is ongoing.
 #include(joinpath(@__DIR__, "src", "generate_validation_table.jl"))
 include(joinpath(@__DIR__, "make_model_library.jl"))
 
 pages = OrderedDict(
         "Welcome Page" => "index.md",
-        "User Guide" => Any[
-            "user_guide/installation.md",
-            "user_guide/quick_start_guide.md",
-            "user_guide/system.md",
-        ],
-        "Modeler" => Any[
-            "modeler/parsing.md",
-            "modeler/data.md",
-            "modeler/example_dynamic_data.md"
-        ],
-        "Model Developer" => Any[
-            "Extending Parsing" => "model_developer/extending_parsing.md",
-            "Adding Types" => "model_developer/adding_types.md",
-        ],
-        "Developer" => Any[
-            "Style Guide" => "developer/style.md",
-            "Testing" => "developer/testing.md",
-            "Logging" => "developer/logging.md",
-        ],
+        "Quick Start Guide" => "quick_start_guide.md",
+        "Tutorials" =>  "tutorials/intro_page.md",
+        "Modeler Guide" =>
+            Any[
+            "modeler_guide/type_structure.md",
+            "modeler_guide/system.md",
+            "modeler_guide/parsing.md",
+            "modeler_guide/time_series.md",
+            "modeler_guide/example_dynamic_data.md",
+            "modeler_guide/data.md",
+            ],
+        "Model Developer Guide" =>
+            Any["Extending Parsing" => "model_developer_guide/extending_parsing.md",
+                "Adding Types" => "model_developer_guide/adding_custom_types.md",
+            ],
+            "Code Base Developer Guide" =>
+            Any["Developer Guide" => "code_base_developer_guide/developer.md",
+            "Adding New Types" => "code_base_developer_guide/adding_new_types.md",
```

Comment:
Shouldn't this be in the `Model Developer` section?

Reply:
I added a comment to that effect